electrumsv/nodeapi.py

```diff
@@ -333,6 +335,16 @@ async def execute_jsonrpc_call_async(request: web.Request, object_data: Any) \
                 error=ErrorDict(code=INVALID_REQUEST,
                     message="Params must be an array or object"))))
 
+    wallet = get_wallet_from_request(request, request_id)
+    if wallet is not None and wallet._network is not None:
+        if not await header_sync_state_middleware(wallet):
+            # Node `JSONRPCRequest::parse` error case.
+            return web.json_response(
```

Comment:
Delete all this and just raise the standard http error.

Reply:
I had to do it this way because of mypy type checking. aiohttp doesn't have a built-in for 425 TooEarly errors and if you raise a `web.json_response` the type checker complains about it.

Reply:
What about this?
```
# aiohttp does not provide this. Define it locally.
class HTTPTooEarly(HTTPClientError):
    status_code = 425
```

Reply:
I tried that already.